### PR TITLE
Zooma mapping updates October 2021

### PR DESCRIPTION
**Pull request by YalanBi**

### CODE REVIEW

File: zoomage_report.CURATED.tsv
Comment (anjaf) [inline zoomage_report.CURATED.tsv:56920]: Cell line mapped to a CHEBI term. That doesn't sound right to me.

File: zoomage_report.CURATED.tsv
Comment (anjaf) [inline zoomage_report.CURATED.tsv:56921]: DEL is an "anaplastic large cell lymphoma" cell line in that experiment. I wouldn't map this to the UBERON term for "deep cell layer".

File: zoomage_report.CURATED.tsv
Comment (anjaf) [inline zoomage_report.CURATED.tsv]: NCBITaxon158046 this accession isn't correct. Needs an underscore.

File: zoomage_report.CURATED.tsv
Comment (anjaf) [inline zoomage_report.CURATED.tsv]: Perhaps "unclassified Rhinovirus"? http://purl.obolibrary.org/obo/NCBITaxon_348531

**Review by anjaf — COMMENTED** [review]
Looks good. A few small issues with an invalid accession. Those trip over the analysis pipeline.

File: zoomage_report.CURATED.tsv
Comment (ngeorgeebi) [reply zoomage_report.CURATED.tsv:56921]: good spot, thanks!

File: zoomage_report.CURATED.tsv
Comment (ngeorgeebi) [reply zoomage_report.CURATED.tsv]: changed, thanks :)

File: zoomage_report.CURATED.tsv
Comment (ngeorgeebi) [reply zoomage_report.CURATED.tsv]: changed, thanks!

File: zoomage_report.CURATED.tsv
Comment (ngeorgeebi) [reply zoomage_report.CURATED.tsv:56920]: row removed, thanks @anjaf

**Comment by ngeorgeebi** [comment]
@anjaf your review would be greatly appreciated

File: zoomage_report.CURATED.tsv
Comment (anjaf) [inline zoomage_report.CURATED.tsv:57329]: I cannot find this term in OLS. Also http://purl.obolibrary.org/obo/FBbt_00058280 doesn't seem to point to any valid term.

File: zoomage_report.CURATED.tsv
Comment (anjaf) [inline zoomage_report.CURATED.tsv:57338]: Same here.

File: zoomage_report.CURATED.tsv
Comment (anjaf) [inline zoomage_report.CURATED.tsv]: And here. Also two different labels mapped to the same term? Is that right? (Just making sure because I can't see the term label for FBbt_00058264)

File: zoomage_report.CURATED.tsv
Comment (anjaf) [inline zoomage_report.CURATED.tsv:57365]: "adult midgut-hindgut hybrid zone" should be http://purl.obolibrary.org/obo/FBbt_00047207

File: zoomage_report.CURATED.tsv
Comment (anjaf) [inline zoomage_report.CURATED.tsv:57400]: This term is missing in OLS too.

File: zoomage_report.CURATED.tsv
Comment (anjaf) [inline zoomage_report.CURATED.tsv]: This is more a comment regarding the curation but also the mapping. What is the numbering here and why is one number in "authors" and the other in "ontology" labels? Should we have these numberings in the "ontology labels" annotation if they are being mapped here to the same CL term?

**Review by anjaf — APPROVED** [review]
This is how far I got for today with the review. Approving now so we can catch the update on Sunday.  

But before you merge, I think it may be problematic with the terms not showing in OLS. Are those newly requested terms? If yes, we might need to wait with the mapping until they are in OLS. Otherwise, it will cause problems with the mapping during analysis.

**Pull request merged**
